src/elements/AddTodoForm.jsx

```diff
@@ -0,0 +1,13 @@
+const AddTodoForm = () => {
+  return (
+    <form>
+      <label htmlFor="todoTitle">Title: </label>
+      <input id="todoTitle" type="text" />
+      <button type="button" id="submitButton">
```

Comment:
Please change the button type from `button` to `submit`. The type of the button used to submit data from a form should be `submit`. A button with the type `button` does not submit form data. It is also possible to omit the button type in the form, and it will default to submit.